MovieGallery/Pages/Shared/_Layout.cshtml

```diff
@@ -22,7 +22,7 @@
                             <a class="nav-link text-dark" asp-area="" asp-page="/Index">Home</a>
                         </li>
                         <li class="nav-item">
-                            <a class="nav-link text-dark" asp-area="" asp-page="/Search">Search by year</a>
+                            <a class="nav-link text-dark" asp-area="" asp-page="/Search">Search</a>
```

Comment:
changed search by year to search in layout file for linking it correctly to search page